imports/wasi_http/wasi_streams/streams.go

```diff
@@ -0,0 +1,84 @@
+package wasi_streams
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/tetratelabs/wazero"
+)
+
+const ModuleName = "streams"
+
+type stream struct {
+	reader io.Reader
+	writer io.Writer
+}
+
+type streams struct {
+	streams            map[uint32]stream
+	stream_handle_base uint32
```

Comment:
Let's change those to camel case to follow Go naming convensions?

Reply:
done. (and elsewhere)